services/web/server/src/simcore_service_webserver/folders/_folders_db.py

```diff
@@ -28,13 +29,25 @@
 _logger = logging.getLogger(__name__)
 
 
+class UnSet:
```

Comment:
I'm confused about the usage of this `UnSet` class. How is it handled by the functions that receive such an object?

Reply:
Since I see it used only in the scope of this module, what's the difference between UnSet and None?

Reply:
It is a standard sentinel commonly used to identify which arguments were set and which not. This is very handy for `update` operations. Please check it online or I can explain you a bit more offline if still confused.